nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/resources/OSGI-INF/notification-subscription-contrib.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<component
+        name="org.nuxeo.ecm.notifications.contribution.comments.autosubscribe.test">
+  <extension target="org.nuxeo.runtime.ConfigurationService"
+    point="configuration">
+    <property name="org.nuxeo.ecm.platform.comment.service.notification.autosubscribe">true</property>
+  </extension>
+</component>
```

Comment:
New line (you probably can configure your IDE to add new line at the end :))

Reply:
need that definitly !